src/core/small_string.h

```diff
@@ -55,7 +57,7 @@ class SmallString {
   char prefix_[kPrefLen];
 
   uint32_t small_ptr_;  // 32GB capacity because we ignore 3 lsb bits (i.e. x8).
-  uint16_t size_;       // uint16_t - total size (including prefix)
+  uint8_t size_;        // uint8_t - total size (including prefix)
```

Comment:
i suggest keeping size_ as `uint16_t` - i have no desire to improve the efficiency of this optimization